firmware/lucidgloves-firmware/src/Encoding/AlphaEncoding.cpp

```diff
@@ -0,0 +1,54 @@
+#include "AlphaEncoding.h"
+
+char* AlphaEncoding::encode(int* flexion, int joyX, int joyY, bool joyClick, bool triggerButton, bool aButton, bool bButton, bool grab, bool pinch, bool calib, bool menu){
+  static char stringToEncode[75];
```

Comment:
ideally this string should created and then passed into the function, instead of it being static inside the function. with this behaviour, the char* value returned will always be the same, so calling encode twice would both point to the same result. Not sure if that's wanted or not?
Same for legacy encoding